base_layer/key_manager/src/cipher_seed.rs

```diff
@@ -264,6 +281,107 @@ impl CipherSeed {
     }
 }
 
+impl CipherSeed {
+    fn generate_hmac(
+        birthday: &[u8],
+        entropy: &[u8],
+        cipher_seed_version: &[u8],
+        salt: &[u8],
+        passphrase: &str,
+    ) -> Result<Vec<u8>, KeyManagerError> {
+        // birthday should be 2 bytes long
+        if birthday.len() != 2 {
```

Comment:
Might be better to avoid the use of a magic number and move this length to a constant, in case it changes in a future version.

Reply:
Nice catch here :) !